src/System.Net.Primitives/ref/System.Net.Primitives.cs

```diff
@@ -54,19 +54,25 @@ public partial class CookieCollection : System.Collections.ICollection, System.C
         public void Add(System.Net.CookieCollection cookies) { }
         public System.Collections.IEnumerator GetEnumerator() { return default(System.Collections.IEnumerator); }
         void System.Collections.ICollection.CopyTo(System.Array array, int index) { }
+        public void CopyTo(Cookie[] array, int index) { }
+        public bool IsReadOnly { get; }
```

Comment:
Also, in the ref assembly, for getter-only properties, is it preferable to:

A) Match existing convention of the file. In this file it's using the `return default(XXX);` convention, e.g.:

```c#
public bool IsReadOnly { get { return default(bool); } }
```

B) Start using the newer `throw null;` convention, e.g.:

```c#
public bool IsReadOnly { get { throw null; } }
```

C) Use getter-only auto-property, e.g.:

```c#
public bool IsReadOnly { get; }
```

?